pkg/network/tracer/tracer_test.go

```diff
@@ -177,6 +192,10 @@ func TestGetStats(t *testing.T) {
 			// DNS stats not supported on some systems
 			continue
 		}
+		if section == "http" && !httpSupported {
```

Comment:
I seem to recall merging this change into `main`, so this may be a conflict or go away when the base branch is rebased.

Reply:
May want to take this out now (dup)

Reply:
without that it fails completely on centos, because the kernel doesn't support it.

Reply:
Sorry I thought it was already there for some reason.